frontend/src/components/dashboard/Dashboard.js

```diff
@@ -217,11 +223,13 @@ export default function Dashboard() {
                     name="Museum Gallery"
                     link="/museum-gallery"
                     isMuseum={true}
+                    artworksError={museumError}
```

Comment:
Instead of passing this as a prop, you could move the `useSelector` to `GalleryPreview` and use a ternary operator and `isMuseum` to get the right error from the reducer.  So in `GalleryPreview` maybe something like:
`const error = useSelector((state) => (isMuseum ?
      state.museumArtworks.error :
      state.userArtworks.error));`